tfgrid3/umbrel/dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:stable-slim
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive\
+    apt-get -qq install wget curl net-tools iputils-ping openssh-server docker.io \
+    fswatch jq rsync sudo iproute2 git gettext-base python3 gnupg avahi-daemon avahi-discover libnss-mdns \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -L https://github.com/docker/compose/releases/download/v2.16.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose && \
+    chmod +x /usr/local/bin/docker-compose
+
+
```

Comment:
Do we need `wget`, `Jq`, `git`, and any unused packages in the image, or all of them are required in the runtime?

Reply:
wget removed but the others required at run time by umbrel.
check this [line](https://github.com/getumbrel/umbrel/blob/d7c9f4d16317d52eef2193458935dc673790f4db/scripts/install#L114)